package.json

```diff
@@ -8,8 +8,8 @@
     "/dist"
   ],
   "bin": {
-    "🚘": "./dist/cli/cli.js",
-    "ipfs-car": "./dist/cli/cli.js"
+    "🚘": "./src/cli/cli.ts",
+    "ipfs-car": "./src/cli/cli.ts"
```

Comment:
Does this work? I would expect this to break the CLI. The bin should point to the node friendly cjs? Users should be able to run `npx ipfs-car` with just node and npm installed.

Reply:
`ts-node` is a devDep so this works. But, it needs to compile to ts everytime, which is not ideal...

The problem here is. If we have:

```js
"bin": {
    "🚘": "./dist/cjs/cli/cli.ts",
    "ipfs-car": "./dist/cjs/cli/cli.ts"
  }
```

And a completely clean repo, when we do `npm install`  the following error happens:

```
ENOENT: no such file or directory, chmod '/Users/vsantos/work/pl/gh/ipfs/ipfs-car/node_modules/ipfs-car/dist/cjs/cli/cli.ts'
```

This is because I am using a package json dep to test `ipfs-car` with imports using it. I can change the imports back

Reply:
done, this was also mostly for me to test it out if the imports were good